Examples/Swift/ViewController.swift

```diff
@@ -242,6 +242,34 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         present(navigationViewController, animated: true, completion: nil)
     }
     
+    func navigationMapView(_ mapView: MGLMapView, viewFor annotation: MGLAnnotation) -> MGLAnnotationView? {
+        guard annotation is MGLUserLocation else { return nil }
```

Comment:
I wonder if this has to do with some of the user location overriding that the navigation SDK does…